cms/djangoapps/models/settings/course_metadata.py

```diff
@@ -246,6 +248,20 @@ def validate_and_update_from_json(cls, descriptor, jsondict, user, filter_tabs=T
                 did_validate = False
                 errors.append({'message': text_type(err), 'model': model})
 
+        # Disallow updates to the proctoring provider after course start
+        proctoring_provider_model = filtered_dict.get('proctoring_provider')
+        if (
+            not user.is_staff and
+            proctoring_provider_model != descriptor.proctoring_provider and
+            datetime.now(pytz.UTC) > descriptor.start
+        ):
+            did_validate = False
+            message = (
+                'The proctoring provider cannot be modified after a course has started.'
+                ' Contact support for assistance'
```

Comment:
I think you want to consider render the contact email address in this message here.

Reply:
good idea, quick question first.  Is the advanced settings page used at all by openedx instances? I imagine we would need to whitelist the email or edX specific text if so.

Reply:
Yeah, you might want to pull out that contact email from the settings. That said, if that is difficult to do, I suggest you can create a new ticket about it and merge this PR as it is.